dsl/pace/dsl/stencil.py

```diff
@@ -187,10 +188,11 @@ def __init__(
 
         if externals is None:
             externals = {}
-
-        stencil_function = gtscript.stencil
-        stencil_kwargs = {**self.stencil_config.stencil_kwargs}
-
+        self.externals = externals
+        self.func = func
+        self.stencil_function = gtscript.stencil
+        self.stencil_kwargs = {**self.stencil_config.stencil_kwargs}
+        self.stencil_object = None
```

Comment:
should we add `self._field_origins`, `self._stencil_run_kwargs` here as well?

Reply:
sure, and _written_fields